### PR TITLE
deps: update opa to v0.24.0

### PR DESCRIPTION
**Pull request by srenatus**
Naively ran `go get -u github.com/open-policy-agent/opa` and `go mod vendor`.

I wonder if it would make sense here to have some automation do this instead? Similar to what happens in https://github.com/open-policy-agent/opa-envoy-plugin...

### CODE REVIEW

**Comment by srenatus** [comment]
Hmm something is wrong here on travis. Any ideas? Have a fallen into some well-known pit?

**Comment by ritazh** [comment]
> I wonder if it would make sense here to have some automation do this instead

@srenatus That's a great idea! Added this [issue](https://github.com/open-policy-agent/frameworks/issues/98) to track

File: constraint/go.mod [hunk 1]
```diff
@@ -3,15 +3,26 @@ module github.com/open-policy-agent/frameworks/constraint
 go 1.13
 
 require (
+	github.com/OneOfOne/xxhash v1.2.8 // indirect
```
Comment (ritazh) [inline constraint/go.mod:6]: Should this PR just focus on updating `github.com/open-policy-agent/opa v0.24.0`?

Comment (srenatus) [reply constraint/go.mod:6]: I might have done something wrong, but that was my intention. I believe this is pulled in as a transitive dependency of OPA, but I can double check in the morning.

Comment (srenatus) [reply constraint/go.mod:6]: https://github.com/open-policy-agent/opa/blob/v0.24.0/go.mod#L6 👈 yeah I think this is correct. It's a dependency of OPA, so upgrading OPA would pull this in.

**Comment by srenatus** [comment]
Looks like travis is happy now 🎉

**Review by maxsmythe — APPROVED** [review]
LGTM

**Comment by maxsmythe** [comment]
@ritazh LGTY?

**Review by ritazh — APPROVED** [review]
lgtm

**Pull request merged**